.golangci.yml

```diff
@@ -0,0 +1,8 @@
+# all available settings of specific linters
```

Comment:
How about removing `complexity` set from linter test? 
https://github.com/klaytn/klaytn/blob/8f3b25f3d8afdc99d5a7441554051572f44f5bda/.circleci/config.yml#L310

Reply:
@aidan-kwon Removed this on another [PR](https://github.com/klaytn/klaytn/pull/1137).